packages/visual-dev/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import styled from 'styled-components'
+
+import { custom } from './Styles.tsx'
```

Comment:
The `Styles` file should only really contain styling related code. For custom SVG icons, it might be worth to instead organise them in a subdirectory called `icons` or something. Basically, we'd just want finding and adding more custom icons to be as easy as possible (without having to scroll through a wall of svg code lol)